bindings/java/src/org/sleuthkit/datamodel/SleuthkitCase.java

```diff
@@ -7877,6 +7916,24 @@ public void updateImagePath(String newPath, long objectId) throws TskCoreExcepti
 	 * @throws TskCoreException
 	 */
 	public Report addReport(String localPath, String sourceModuleName, String reportName) throws TskCoreException {
+		return addReport(localPath, sourceModuleName, reportName, null);
+	}
+
+	/**
+	 * Inserts a row into the reports table in the case database.
+	 *
+	 * @param localPath        The path of the report file, must be in the
+	 *                         database directory (case directory in Autopsy) or
+	 *                         one of its subdirectories.
+	 * @param sourceModuleName The name of the module that created the report.
+	 * @param reportName       The report name, may be empty.
```

Comment:
See comment above - I'm not sure why I thought empty string report names were a good idea.

Reply:
Are you simply looking for a comment update here too or do you want us to enforce it?
If we choose to enforce it, do we want to change the schema to make in a non null column?

Reply:
The schema already specifies "report_name TEXT NOT NULL." The "may be empty" in the parameter comments is literally true - a NULL argument is an error, but the empty string is legal. I don't think we should necessarily get into generally more rigorous method parameter checking as a policy at this point in the game, but I would rather not openly suggest in the comments in this class that the empty string is okay as a report name.

Reply:
Ok, the updated comment will be taken care of in a subsequent pull request. If that's everything can you go ahead and take this pull request (and the related Autopsy PR) and the requested changes will be included in the "search" PR?

Reply:
Okay.